File: csharp/darts/Darts.cs
using System;

public static class Darts
{
    public static int Score(double x, double y)
    {
        double distance = Math.Sqrt(x * x + y * y);

        return 
            distance <= 1 ? 10 : 
            distance <= 5 ? 5 : 
            distance <= 10 ? 1 : 
            0;
    }
}

File: java/darts/src/main/java/Darts.java
class Darts {

    private final double distance;

    Darts(double x, double y) {
        distance = Math.sqrt(x*x+y*y);
    }

    int score() {
        if(distance <=  1d) return 10;
        if(distance <=  5d) return  5;
        if(distance <= 10d) return  1;

        return 0;
    }

}
